// django-angular/django.umd.js
(function (global, factory) {
	typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core'), require('@angular/common'), require('@angular/common/http')) :
	typeof define === 'function' && define.amd ? define(['exports', '@angular/core', '@angular/common', '@angular/common/http'], factory) :
	(factory((global.django = {}),global.core,global.common,global.http));
}(this, (function (exports,core,common,http) { 'use strict';

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes} checked by tsc
 */
var DjangoModule = (function () {
    function DjangoModule() {
    }
    /**
     * @return {?}
     */
    DjangoModule.forRoot = /**
     * @return {?}
     */
    function () {
        return {
            ngModule: DjangoModule,
            providers: []
        };
    };
    DjangoModule.decorators = [
        { type: core.NgModule, args: [{
                    imports: [
                        common.CommonModule,
                        http.HttpClientModule,
                    ],
                    declarations: [],
                    exports: []
                },] },
    ];
    return DjangoModule;
}());

exports.DjangoModule = DjangoModule;

Object.defineProperty(exports, '__esModule', { value: true });

})));
